docs/howto/index.md

```diff
@@ -62,11 +63,6 @@ Instructions how to get tasks done with CrateDB.
 - {ref}`terraform-usage`
 - {ref}`trino-usage`
 
-:::{rubric} 2022
-:::
-
-- {ref}`prometheus-usage`
-
```

Comment:
Refreshing this article did not take many efforts. It was merely just copy/paste from a series of others following the same scheme, then slotting in the Docker Compose file and configuration files from the original article. If you agree with those kinds of changes, we'd like to encourage you to find similar spots to modernize easily.